firmware/makefile

```diff
@@ -114,6 +140,9 @@ OBJ_DIR = obj
 
 # This is a list of objects to build
 OBJECTS=$(patsubst %.c, $(OBJ_DIR)/%.o, $(SOURCES))
+ifdef ESP_GDB
+OBJECTS += obj/gdbstub/gdbstub-entry.o
```

Comment:
Is this not captured in `SOURCES` already?

Reply:
Ask Richard. He told me it was quite a fiddle to get gdb set up

Reply:
I'll fiddle around with the makefile until I like it, don't worry about it.

Reply:
It was an S file, not a c file, which is why this was the way it was. I added a scan for S files instead